include/oneapi/ccl/config.h

```diff
@@ -32,10 +32,10 @@
 
```

Comment:
@nikitaxgusev I have asked this question over another channel - do we really want to version this file ? It's generated on every rerun and not versioned in innersource.

Reply:
removed